src/oic/utils/keyio.py

```diff
@@ -451,8 +457,8 @@ def __init__(
             ``requests`` documentation.
         :return:
         """
-        self.spec2key = {}
-        self.issuer_keys = {}
+        self.spec2key = {}  # type: ignore  # FIXME: This seems unused -> Deprecate?
```

Comment:
yes, deprecate seems ok

Reply:
Couldn't find a use so I removed it completely.